docs/Manage-Space.rst

```diff
@@ -261,35 +264,14 @@ follows:
    -  Close: The user cannot request for joining, but only the space
       manager can invite him.
 
-4. In the **Invite users from group** tab, select all members of a specific
-group for your space where you are already 'manager'.
-
-One of the two following cases occurs when you select this tab.
-
--  **The first instance**: You have been already the manager of a group. It means that you have created at least a space.
-
-   |image24|
-
-   -  **i.** Tick the **Select a group of people** to invite in your space checkbox to open the form below.
-   
-   |image32|
-
-   -  **ii.** Select the group in the left pane, then its child group in
-      the right pane.
-
-      You will see your selected group as below.
-
-      |image25|
-
-.. note:: To remove the selected groups, untick the **Select a group of people to invite in your space** checkbox.
+5. In the **Users ** field, the space creator can invite users or space members to the space to create just by picking them with the selector.
 
-   After you have selected one existing group, all users in that group will be invited to join your newly created space.
+Type-ahead suggestions allow to facilitate selection users and groups to invite.
 
--  **The second instance**: You have not been a manager of any group yet. It means that this is the first time you create a space, so you cannot select a target group.
+.. note:: Only spaces that the creator is member of can be selected.
 
-   |image26|
 
-5. Click Create Space button to finish adding your new space. The new space appears.
+6. Click Create Space button to finish adding your new space. The new space appears.
```

Comment:
@hghorbel  we should have some style  for representing product labels such as button names. @mzorai what do you suggest ?

Reply:
we can use these quotes ``Create Space``, they will appear in red

Reply:
looks like the expression was interpreted. In fact we use the quotes `` (with alt gr + 7 of the keyborad)